handlers.py

```diff
@@ -1,9 +1,19 @@
 import kopf
 import kubernetes
-
-
-@kopf.on.create('', 'v1', 'secrets', annotations={'synator/sync': 'yes'})
-@kopf.on.update('', 'v1', 'secrets', annotations={'synator/sync': 'yes'})
+import os
+
+def watch_namespace(namespace, **_):
+    namespaces = os.getenv('WATCH_NAMESPACE')
```

Comment:
Could you set this variable out of function? We don't need to get env every time

Reply:
Can you please verify?